CoroutineTemplate/app/src/main/res/values/colors_pallete.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Declaring colors with their real name from the style guide we get -->
+    <color name="free_speech_blue">#3F51B5</color>
+    <color name="tory_blue">#303F9F</color>
+    <color name="violet_red">#FF4081</color>
```

Comment:
yes, but I think we should put the main color at first, e.g. red_violet, red_rose 😂

Reply:
omg 🙈

Reply:
Done ✅ ✅ ✅ ✅